ONSdigital/ssdc-rm-docker-dev
# Adding print file service

# Motivation and Context

Added print file service to the docker-compose file

# What has changed



- added printfile service to the docker compose file
- Updated SFTP key

# How to test?



- Run make up with [New print file service built](https://github.com/ONSdigital/ssdc-rm-print-file-service/pull/1)
- Create a survey and collex 
- upload sample
- create wave of contact
- Should see it go through the new print file service

# Links



[Trello](https://trello.com/c/WmNgESQo/)
